src/components/routes/PrivateSwitch.js

```diff
@@ -3,13 +3,30 @@ import { Switch } from 'react-router-dom';
 import { bool } from 'prop-types';
 
 import routes from 'constants/routesPaths';
+import CreateTargetForm from 'components/common/Home/CreateTargetForm';
 import PrivateRoute from './PrivateRoute';
 import HomePage from '../../pages/HomePage';
+import HomeInfo from '../../pages/HomePage/HomeInfo';
+import Landing from '../common/Home/Landing';
 
 const PrivateSwitch = ({ authenticated }) => {
   return (
     <Switch>
-      <PrivateRoute path={routes.index} component={HomePage} authenticated={authenticated} />
+      <HomePage>
```

Comment:
What do you think about renaming that `<HomePage>` component to `HomePageLayout`? It looks a little confusing to me having nested pages in it 🤔